### ISSUE TITLE
How to handle the Remote notification in foreground state ?

### ISSUE THREAD

**Issue by hardikmehta88999** [react-native-push-notification/ios#373]
Description: The push notification is coming to the real device when the app is in the foreground state but i don't know how to handle that notification in react native.

**Comment by ShubhamBathe-KM** [comment]
@hardikmehta88999 You need to add some code in your AppDelegate.m file under iOS folder:-

**Comment by Zernach** [comment]
Yes, I also followed those steps in the documentation, but my app's notifications still do not appear at the top of the screen if the notification was received while the app is open in the foreground. 😪

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/54771184/198108111-eb363b70-0b6b-4d86-860c-13781369d1e4.png">

<img width="1375" alt="image" src="https://user-images.githubusercontent.com/54771184/198108259-68d157d7-8944-4515-a9ed-0eeef410e573.png">

**Comment by AnHoang13** [comment]
check if ``onNotification`` return data, then based on that data you can create a local push notification
